server/index.js

```diff
@@ -1,10 +1,105 @@
 // Entry point of express server
 const express = require('express');
+const cors = require('cors');
+const { Pictures, ReviewPhotos } = require('../mongoDB/index');
+
 const app = express();
 const port = 3000;
 
-app.use(express.static(__dirname + '/../public'))
+app.use(cors());
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+app.use(express.static(`${__dirname}/../public`));
+
+// GET request for photos corresponding to a item
+app.get('/pictures/itemId/:itemId', (req, res) => {
+  const { itemId } = req.params;
+  Pictures.find({ item_id: itemId })
+    .then((item) => {
+      const {
+        // eslint-disable-next-line camelcase
+        item_id, store_id, item_pictures, seller_picture, store_picture,
+      } = item[0];
+      const response = {
+        item_id,
+        store_id,
+        item_pictures,
+        seller_picture,
+        store_picture,
+      };
+      res.status(200).json(response);
+    })
+    .catch((err) => {
+      res.status(400).send('Error:', err);
+    });
+});
+
+// GET request for a list of all items that belong to a store
+app.get('/pictures/storeId/:storeId', (req, res) => {
+  const { storeId } = req.params;
+  Pictures.find({ store_id: storeId })
+    .then((items) => {
+      const response = items.map((item) => {
+        const {
+          // eslint-disable-next-line camelcase
+          item_id, store_id, item_pictures, seller_picture, store_picture,
+        } = item;
+        return ({
+          item_id, store_id, item_pictures, seller_picture, store_picture,
+        });
+      });
+      res.status(200).json(response);
+    })
+    .catch((err) => {
+      res.status(400).send('Error:', err);
```

Comment:
if im a consumer making a request and accidentally add the wrong store number or character, this won't be a very helpful message for me

Reply:
Agree with Nick for the long haul but since we're taking an iterative approach to this (or at least I am) I like that you're handling errors at all to start with. :)